src/main/scala/org/constellation/DAO.scala

```diff
@@ -398,13 +403,20 @@ class DAO(
       boundedExecutionContext
     )
 
+    missingProposalFinder = MissingProposalFinder(
+      ConfigUtil.constellation.getInt("snapshot.snapshotHeightInterval"),
+      ConfigUtil.constellation.getLong("snapshot.missingProposalOffset"),
+      ConfigUtil.constellation.getLong("snapshot.missingProposalLimit"),
+      id
+    )
```

Comment:
if there are no dependencies, I'd suggest you to create a val from it, not var and then assign

Reply:
Ok, I will try